www/include/core/header/htmlHeader.php

```diff
@@ -44,75 +45,140 @@
 
 print "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n";
 
-?>
+$cssThemes = "Centreon-Light";
+$userId = (int) $centreon->user->user_id;
+$statement = $pearDB->prepare('SELECT contact_theme FROM contact WHERE contact_id = :contactId');
+$statement->bindValue(':contactId', $userId, \PDO::PARAM_INT);
+$statement->execute();
+if ($result = $statement->fetch(\PDO::FETCH_ASSOC)) {
+    switch ($result['contact_theme']) {
+        case 'light':
+            $cssThemes = "Centreon-Light";
+            break;
+        case 'dark':
+            $cssThemes = "Centreon-Dark";
+            break;
+        default:
```

Comment:
Use the theme name directly.
Be careful, for security reasons, please check that the name follows the following regex: ^([0-9a-zA-Z-]+)$